test/cases/configuration/test_configuration.py

```diff
@@ -32,8 +32,8 @@ def test_no_config(self):
         # implied.
         self.get_default_config()
 
-    def run_explicit_config_test(self, conf_relative_path):
-        conf_path = Path(__file__).parent / conf_relative_path
+    def test_in_http(self):
+        conf_path = Path(__file__).parent / 'conf' / 'in_http.conf'
```

Comment:
There's a couple of different styles of building conf_path in this file, can they be applied consistently?
(Also I was briefly confused about the division going on. Damn operator overloading)